docs/building-your-quest/triggers-and-payloads.md

```diff
@@ -35,6 +35,14 @@ Generated payload:
 
 - `userMessageText`: the text the user wrote to the bot.
 
+### GITHUB_PUSH
+
+Triggered when the user push something to their repository
+
+Generated payload:
+
+- `githubPushHeadCommit`: the body of the relevant commit that was pushed.
```

Comment:
Can you share the engine PR?

Reply:
I didn't add anything - it was always there, I just added the documentation

Reply:
Ok